assets/amqpstreaming/lightning/index.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+		<title>reveal.js</title>
+
+		<link rel="stylesheet" href="css/reveal.css">
+		<link rel="stylesheet" href="css/theme/black.css">
+
+		<!-- Theme used for syntax highlighting of code -->
+		<link rel="stylesheet" href="lib/css/zenburn.css">
+
+		<!-- Printing and PDF exports -->
+		<script>
+			var link = document.createElement( 'link' );
+			link.rel = 'stylesheet';
+			link.type = 'text/css';
+			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
+			document.getElementsByTagName( 'head' )[0].appendChild( link );
+		</script>
+	</head>
+	<body>
+		<div class="reveal">
+			<div class="slides">
+				<section>
+                  <h1>AMQP - Spark Streaming</h1>
+                  real time "temperature" analysis
+                </section>
+				<section>
+                  <h2>What does it do?</h2>
+                  <ul>
+                    <li class="fragment">An AMQP producer sends "temperature" data to a queue</li>
+                    <li class="fragment">A Spark driver gets this data through AMQP</li>
+										<li class="fragment">Max temperature value in the last 5 seconds is shown</li>
+                  </ul>
```

Comment:
this is only 2 slides, it might be nice to have a little more material, but at the least please add a final slide with links to the upstream project. i think that would at least give viewers a clear idea of where to go for more information.